src/stats.jl

```diff
@@ -379,8 +258,8 @@ function summarystats(
     # Subset the chain.
     _chains = Chains(chains, _clean_sections(chains, sections))
 
-    # Caluclate ESS separately.
-    ess_df = ess(_chains; sections = nothing, maxlag = maxlag)
+    # Calculate ESS separately.
+    ess_df = ess(_chains; sections = nothing, maxlag = maxlag) # this is a ChainDataFrame - maybe sections = _default_sections() by default
```

Comment:
Ah we actually already pass `sections = nothing` :smiley:

Reply:
Regarding your comment, why would you want to use `sections = default_sections()` instead?